cdap-validation/bin/test_helpers.py

```diff
@@ -0,0 +1,84 @@
+import os
+import base64
+import urllib2
+import json
+import sys
+
+def usage():
+    print "\nThis is the usage function\n"
```

Comment:
not a useful message. Usage needs a lot more info. I have no idea how I'm supposed to use this